examples/examples/event_error_details.rs

```diff
@@ -59,16 +59,14 @@ async fn main() {
 	println!("Sending an extrinsic from Alice (Key = {}),\n\nto Bob (Key = {})\n", alice, bob);
 	println!("[+] Composed extrinsic: {:?}\n", xt);
 
-	// Send and watch extrinsic until Ready.
-	let _tx_hash = api.submit_and_watch_extrinsic_until(xt, XtStatus::Ready).unwrap();
+	// Send and watch extrinsic until InBlock.
+	let result = api.submit_and_watch_extrinsic_until_success(xt, false);
 	println!("[+] Transaction got included into the TxPool.");
 
 	// Subscribe to system events. We expect the transfer to fail as Alice wants to transfer all her balance.
```

Comment:
Update the comment. I think just removing 'Subscribe to system events.' would be enough

Reply:
Good catch, thank s !

I updated the comments - is it clearer now?